src/routes/console/project-[project]/settings/smtp/+page.svelte

```diff
@@ -0,0 +1,154 @@
+<script lang="ts">
+    import { CardGrid, Heading } from '$lib/components';
+    import { Button, Form, FormList, InputText, InputChoice } from '$lib/elements/forms';
+    import { Container } from '$lib/layout';
+    import { project } from '../../store';
+    import { onMount } from 'svelte/internal';
+    import InputPassword from '$lib/elements/forms/inputPassword.svelte';
+    import { sdk } from '$lib/stores/sdk';
+    import { invalidate } from '$app/navigation';
+    import { Dependencies } from '$lib/constants';
+    import { addNotification } from '$lib/stores/notifications';
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import InputNumber from '$lib/elements/forms/inputNumber.svelte';
+    import { base } from '$app/paths';
+    import { deepEqual } from '$lib/helpers/object';
+
+    let enabled = false;
+    let sender: string;
+    let host: string;
+    let port: number;
+    let username: string;
+    let password: string;
+
+    onMount(() => {
+        enabled = $project.smtpEnabled ?? false;
+        sender = $project.smtpSender;
+        host = $project.smtpHost;
+        port = $project.smtpPort;
+        username = $project.smtpUsername;
+        password = $project.smtpPassword;
+    });
+
+    async function updateSmtp() {
+        const path = '/projects/' + $project.$id + '/smtp';
+
+        const params = {
+            enabled,
+            sender,
+            host,
+            port,
+            username,
+            password
+        };
+        try {
+            await sdk.forConsole.client.call(
```

Comment:
This should be fixed once we add the method to the SDK right?

Reply:
@ArmanNik, I don't think so because using the SDK vs manually making the API call doesn't impact the error message. It's the server that should return a more helpful error.